Resources/views/frontend/checkout/confirm.tpl

```diff
@@ -19,8 +19,7 @@
 {block name='frontend_index_body_attributes'}
     {$smarty.block.parent}
     data-adyenAjaxDoPaymentUrl="{url module='frontend' controller='adyen' action='ajaxDoPayment'}"
-    data-adyenAjaxIdentifyShopperUrl="{url module='frontend' controller='adyen' action='ajaxIdentifyShopper'}"
-    data-adyenAjaxChallengeShopperUrl="{url module='frontend' controller='adyen' action='ajaxChallengeShopper'}"
+    data-adyenAjaxThreeDsUrl="{url module='frontend' controller='adyen' action='ajaxThreeDs'}"
```

Comment:
In order to implement the controller generically we are deprecating the ajaxThreeDs function from this controller in favor of the paymentDetails function, if we change that here as well does it still work?

Reply:
If the template is updated the function will remain the same